frontend/src/pages/org/workflows-list.ts

```diff
@@ -2,41 +2,51 @@ import type { HTMLTemplateResult, PropertyValueMap } from "lit";
 import { state, property, query } from "lit/decorators.js";
 import { msg, localized, str } from "@lit/localize";
 import { when } from "lit/directives/when.js";
-import debounce from "lodash/fp/debounce";
-import flow from "lodash/fp/flow";
-import map from "lodash/fp/map";
-import orderBy from "lodash/fp/orderBy";
-import filter from "lodash/fp/filter";
-import Fuse from "fuse.js";
+import { ifDefined } from "lit/directives/if-defined.js";
 import queryString from "query-string";
 
 import type { AuthState } from "../../utils/AuthService";
 import LiteElement, { html } from "../../utils/LiteElement";
 import type { Crawl, Workflow, WorkflowParams } from "./types";
 import { CopyButton } from "../../components/copy-button";
 import { SlCheckbox } from "@shoelace-style/shoelace";
-import type { APIPaginatedList } from "../../types/api";
+import type { APIPaginatedList, APIPaginationQuery } from "../../types/api";
+import type { PageChangeEvent } from "../../components/pagination";
 
-type SortField = "_lastUpdated" | "_name";
+type SearchFields = "name" | "firstSeed";
+type SortField = "lastRun" | "name" | "firstSeed" | "created" | "modified";
 type SortDirection = "asc" | "desc";
 
 const FILTER_BY_CURRENT_USER_STORAGE_KEY =
   "btrix.filterByCurrentUser.crawlConfigs";
-const INITIAL_PAGE_SIZE = 30;
+const INITIAL_PAGE_SIZE = 10;
 const POLL_INTERVAL_SECONDS = 10;
-const MIN_SEARCH_LENGTH = 2;
+const ABORT_REASON_THROTTLE = "throttled";
+
 const sortableFields: Record<
   SortField,
   { label: string; defaultDirection?: SortDirection }
 > = {
-  _lastUpdated: {
+  lastRun: {
     label: msg("Last Updated"),
     defaultDirection: "desc",
   },
-  _name: {
-    label: msg("Name"),
+  firstSeed: {
+    label: msg("Crawl Start URL"),
+    defaultDirection: "asc",
+  },
+  name: {
+    label: msg("Workflow Name"),
     defaultDirection: "asc",
   },
+  created: {
+    label: msg("Workflow Created"),
+    defaultDirection: "desc",
+  },
+  modified: {
+    label: msg("Workflow Edited"),
```

Comment:
```suggestion
    label: msg("Last Updated"),
```

I guess would go either way with edited vs updated?